tools/dynamatic/dynamatic.cpp

```diff
@@ -639,20 +640,31 @@ int main(int argc, char **argv) {
       return 1;
     }
 
-    // Read the script line-by-line and execute its commands
+    // Read the script line-by-line and execute its commands supported
+    // delimeters: '\n' and ';'
     std::string line;
-    while (std::getline(inputFile, line))
-      if (!line.empty() && !StringRef(line).starts_with("#")) {
-        llvm::outs() << PROMPT << line << "\n";
-        handleInput(line);
-      }
+    while (std::getline(inputFile, line, '\n')) {
+      if (StringRef(line).starts_with("#"))
+        continue;
+      // cast '\n'-separated lines into streams, then split it by ';'
+      std::stringstream sLine(line);
+      for (std::string cmd; getline(sLine, cmd, ';');)
```

Comment:
nit: the style convention here is to put brackets around the for loop since its body spans multiple lines (even if it's only one statement). In the early days of the repo I didn't respect this myself but the LLVM style conventions prohibit this, so now I also put brackets in these situations; I think it improves readability slightly, which is the LLVM rationale.